.github/workflows/ui.yml

```diff
@@ -53,13 +53,13 @@ jobs:
         run: yarn build:${{ inputs.environment }}
 
       - name: Get AWS Creds
-        if: github.actor != 'dependabot[bot]'
+        # if: github.actor != 'dependabot[bot]'
```

Comment:
I disabled the dependabot PR feature. It will still alert CMS cloud security, but it will not submit PR fixes so we can delete these lines. keep code clean